data/RWTH-NFDI-Jupyter-Service.yaml

```diff
@@ -0,0 +1,42 @@
+title: RWTH Aachen University
+provider: HPC.ITC.RWTH
+service_url: https://jupyterhub.hpc.itc.rwth-aachen.de:9651/
+support: servicedesk@itc.rwth-aachen.de
+health_api_url: https://jupyterhub.hpc.itc.rwth-aachen.de:9651/hub/api/health
+documentation_url: https://help.itc.rwth-aachen.de/service/rhr4fjjutttf/article/689934fec5a34c909c54606f6bc2e827/
+target_group_open_for: Researchers, RWTH Students/Employees/Guests, Juelich Members, Users sponsored by other Users, Users upon project application 
+login_process: Login with RWTH HPC Account through most NHR or NRW VPNs
+features:
+  version: JupyterHub 3.0+, JupyterLab 3.4.7+
+  kernels: ["Python 3.9", "Octave", "R", "C++ 11/14/17", "Matlab 2022b", "Matlab 2020b",  "Python: Keras", "Python: Tensorflow",  "Python: Tensorboard", "Python: Gurobi", "Python: Pytorch" ]
+  extensions: ["ipytree",
+        "jupyterlab-drawio",
+        "jupyterlab-plotly",
+        "jupyterlab_iframe",
+        "nbdime-jupyterlab",
+        "jupyter-matplotlib",
+        "jupyterlab_pygments",
+        "jupyterlab/git",
+       " jupyterlab/latex",
+        "jupyter-widgets/jupyterlab-manager"]
+  proxy_apps: ["Matlab 2022b", "Matlab 2020b"]
+  install: false
+  shared_folder: true
+  persistent_storage: true
+  misc: ["Available core-hours vary based on user affiliation or project.", "Runs on HPC Hardware", "Users installs planned, by request only at the moment."]
+resources:
+  default_server_user: 1
+  max_server_user: 5
+  # cpu values refer to vCPUs and are guarantees in case of contention, limits are 8x higher
```

Comment:
I thought about another column here, named "burst" (see #1) for the limits - would that help to better reflect the actual setup?

Reply:
Our system allocates resources exclusively in accordance to our HPC systems policy. There can be up to 5 servers per user running in the cluster. The existing fields suffice for us.

Reply:
Github always shows 4 lines for a comment, although I only selected on line. My comment refers to your comment: `# cpu values refer to vCPUs and are guarantees in case of contention, limits are 8x higher`

Reply:
I tried to do a mapping here that best reflects our circumstances. Our cpus are fully guaranteed and are physical. The limits are defined by the user on submissions up to the maximum (48 cpus per node). There is no contention. Not sure if this is the intent.